tools/ci_build/github/linux/docker/Dockerfile.manylinux2014_gpu

```diff
@@ -25,14 +25,14 @@ ENV SSL_CERT_FILE=/opt/_internal/certs.pem
 
 #Build manylinux2014 docker image end
 
-ARG PYTHON_VERSION=3.5
-ARG BUILD_EXTR_PAR
+ARG PYTHON_VERSION=3.6
+ARG INSTALL_DEPS_EXTRA_ARGS
```

Comment:
As we plan to remove python 3.5 support soon, this is fine.

Reply:
Wait, why do we need this argument in the first place? Should the image support all the python versions?

Reply:
Oh, it is only for training.

Reply:
it's a manylinux image with multiple python versions, but this parameter is used to set the python version for which the dependencies are installed.
yes, this one is only for training. ideally at some point we can use the same image